CHANGELOG.md

```diff
@@ -1,5 +1,105 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.37
+
+<!-- generated comparing v5.0.0-alpha.36..next -->
+
+_Jun 15, 2021_
+
+Big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚒️ We've introduced a new `useTheme` and `useThemeProps` hooks in the `@material-ui/system` package.
+  We believe that this package can be used as a standalone styling solution for building custom design systems (#26649) @mnajdova.
+- 💥 Made progress with the breaking changes. We have done 105 of the 109 changes [planned](https://github.com/mui-org/material-ui/issues/20012). We are getting closer to our goal of releasing 5.0.0-beta.0 on July 1st and start to promote its usage over v4. You can also follow [our milestone](https://github.com/mui-org/material-ui/milestone/35) for more details.
+- And many more 🐛 bug fixes and 📚 improvements.
+
+### `@material-ui/core@5.0.0-alpha.37`
+
+#### Breaking changes
+
+- <!-- 10 --> [Button] Remove label span (#26666) @siriwatknp
+
+The `span` element that wraps children has been removed. `label` classKey is also removed.
+
+```diff
+<button class="MuiButton-root">
+- <span class="MuiButton-label">
+    children
+- </span>
+</button>
+```
+
+#### Changes
+
+- <!-- 08 --> [Button] Add missing color type (#26593) @sakura90
+- <!-- 07 --> [Grid] Add responsive direction and spacing props (#26590) @likitarai1
```

Comment:
We probably want to emphasize on this change, with a quick demo on the changelog.

Reply:
Updated, please take a look.